test/cli/iGenVar_cli_test.cpp

```diff
@@ -103,54 +100,107 @@ std::string expected_res
     "CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\n"
 };
 
-TEST_F(detect_breakends, no_options)
+std::string expected_err_default_no_err
+{
+    "INS: chr21\t41972615\tForward\tchr21\t41972616\tForward\t1681\tm2257/8161/CCS\n"
+    "BND: chr22\t17458417\tForward\tchr21\t41972615\tForward\t2\tm41327/11677/CCS\n"
+    "BND: chr22\t17458418\tForward\tchr21\t41972616\tForward\t0\tm21263/13017/CCS\n"
+    "BND: chr22\t17458418\tForward\tchr21\t41972616\tForward\t0\tm38637/7161/CCS\n"
+    "Start clustering...\n"
+    "Done with clustering. Found 3 junction clusters.\n"
+    "No refinement was selected.\n"
+};
+
+TEST_F(iGenVar_cli_test, no_options)
 {
     cli_test_result result = execute_app("iGenVar");
-    std::string expected
+    std::string expected_res
```

Comment:
This local variable has the same name as the global variable above. Maybe rename to `expected_res_no_options`?

Reply:
Or we move the global variable into the test cases which use it? I guess we could decide to either define all result and error messages above the test cases or inside each test case. A mix of both is confusing.